reference/3.0/Microsoft.PowerShell.Core/Invoke-Command.md

```diff
@@ -1151,49 +1173,61 @@ returns the output of the invoked command, which is the value of the **ScriptBlo
-  from all sessions. It might be connected to a different session. To determine whether you can
-  connect or reconnect to the session, use the **Availability** property.
-
-  An **Availability** value of None indicates that you can connect to the session. A value of Busy
-  indicates that you cannot connect to the PSSession because it is connected to another session.
-
-  For more information about the values of the **State** property of sessions, see
-  [RunspaceState Enumeration](https://msdn.microsoft.com/library/system.management.automation.runspaces.runspacestate).
-
-  For more information about the values of the **Availability** property of sessions, see
-  [RunspaceAvailability Enumeration](https://msdn.microsoft.com/library/system.management.automation.runspaces.runspaceavailability).
+On Windows Vista, and later versions of the Windows operating system, to use the **ComputerName**
+parameter of `Invoke-Command` to run a command on the local computer, you must open PowerShell by
+using the Run as administrator option.
+
+When you run commands on multiple computers, PowerShell connects to the computers in the order in
+which they appear in the list. However, the command output is displayed in the order that it's
+received from the remote computers, which might be different.
+
+Errors that result from the command that `Invoke-Command` runs are included in the command results.
+Errors that would be terminating errors in a local command are treated as non-terminating errors in
+a remote command. This strategy makes sure that terminating errors on one computer don't close the
+command on all computers on which it's run. This practice is used even when a remote command is run
+on a single computer.
+
+If the remote computer isn't in a domain that the local computer trusts, the computer might not be
+able to authenticate the user's credentials. To add the remote computer to the list of trusted hosts
+in WS-Management, use the following command in the `WSMAN` provider, where `<Remote-Computer-Name>`
+is the name of the remote computer:
+
+`Set-Item -Path WSMan:\Localhost\Client\TrustedHosts -Value \<Remote-Computer-Name\>`
+
+In PowerShell 2.0, you can't use the `Select-Object` cmdlet to select the **PSComputerName**
+property of the object that `Invoke-Command` returns. Instead, to display the value of the
+**PSComputerName** property, use the dot method to get the **PSComputerName** property value
+($result.PSComputerName), use a **Format** cmdlet, such as the `Format-Table` cmdlet, to display the
+value of the **PSComputerName** property, or use a `Select-Object` command where the value of the
+property parameter is a calculated property that has a label other than PSComputerName. This
+limitation doesn't apply to PowerShell 3.0 or later versions.
+
+When you disconnect a **PSSession**, such as by using **InDisconnectedSession**, the session state
```

Comment:
Change to:

When you disconnect a **PSSession**, by using the **InDisconnectedSession** parameter, the session state

Reply:
Fixed